extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -54,9 +54,9 @@
     CLASS_CACHE_KEY = ByteBuffer.allocate(keyUtf8.length + 1).put(keyUtf8).put((byte) 0xFF).array();
   }
 
-  private volatile boolean started = false;
+  CacheScheduler.Entry entry = null;
```

Comment:
The example in https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/locks/ReentrantReadWriteLock.html uses `volatile boolean cacheValid` which makes me think this should also be `volatile`